src/snowflake/cli/plugins/nativeapp/commands.py

```diff
@@ -148,7 +148,7 @@ def app_open(
     **options,
 ) -> CommandResult:
     """
-    Opens the application inside of your browser,
+    Opens the application object inside of your browser,
```

Comment:
Here, the application is a same-account installation. Hence using "application object" term. The Open command will open up the Snowsight link to this application, not the worksheet. 
Regardless same account or cross account install, the application is always listed under Apps in Snowsight.
So should this be "application object" or "Snowflake Native App"?

Reply:
This is a problem with the terminology we were give (and something I'll address in my proposal). I think it should be Snowflake Native App because it's in the App tab under Snowsight (unless they changed the UI again).